ambari-server/src/test/java/org/apache/ambari/server/agent/TestHeartbeatHandler.java

```diff
@@ -989,7 +980,7 @@ public void testRecoveryStatusReports() throws Exception {
           add(command);
         }}).anyTimes();
     replay(am);
-    HeartBeatHandler handler = new HeartBeatHandler(fsm, am, injector);
+    HeartBeatHandler handler = new HeartBeatHandler(fsm, am, Encryptor.NONE, injector);
```

Comment:
how about adding some method like getDefaultHeartBeatHandler(am) so we call it instead of a copy paste with 4 params, 3 of which do not really matter?

Reply:
yeah. good idea, originally i was going to do that but i forgot